xchange-bitmax/src/main/java/org/knowm/xchange/bitmax/service/BitmaxAccountServiceRaw.java

```diff
@@ -0,0 +1,27 @@
+package org.knowm.xchange.bitmax.service;
+
+import org.knowm.xchange.Exchange;
+import org.knowm.xchange.bitmax.BitmaxException;
+import org.knowm.xchange.bitmax.dto.account.BitmaxCashAccountBalanceDto;
+
+import java.io.IOException;
+import java.util.List;
+
+public class BitmaxAccountServiceRaw extends BitmaxBaseService {
+
+  public BitmaxAccountServiceRaw(Exchange exchange) {
+    super(exchange);
+  }
+
+  public List<BitmaxCashAccountBalanceDto> getBitmaxCashAccountBalance() throws IOException {
+    try {
+      return checkResult(
+          bitmaxAuthenticated.getCashAccountBalance(
+              exchange.getExchangeSpecification().getApiKey(),
+              exchange.getNonceFactory().createValue(),
+              signatureCreator));
+    } catch (IOException e) {
+      throw new BitmaxException(e.getMessage());
```

Comment:
we don't wrap IOExceptions

place take a look if you could make the BitmaxException deserializable this response:
https://bitmax-exchange.github.io/bitmax-pro-api/#error-message
and make it thrown by the API interface

place also take a look how error handing is done in binance as described here:
https://github.com/knowm/XChange/wiki/New-Implementation-Best-Practices#step-7

Reply:
Can you please make the merge and we can fix it later. It is not the end of the world, with this implementation. Thanks

Reply:
Please fix it first. Otherwise it will most likely get forgotten...

Reply:
I think that i have fix it. Please check.Thanks